pkg/apis/aadpodidentity/types.go

```diff
@@ -0,0 +1,228 @@
+package aadpodidentity
+
+import (
+	api "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
```

Comment:
Is 'internal' directory within the aadpodidentity more appropriate for these files ?

Reply:
golang has restrictions on how directories with name internal are used, left as is